api/src/main/java/resources/META-INF/LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
+                           Version 2.0, January 2004
+                        http://www.apache.org/licenses/
```

Comment:
This file is located in `src/main/java/resources` as if it's to be added to a jar, but the packaging type for this artifact is `pom`

Reply:
This still needs removed